Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:current-alpine
+WORKDIR /app
+COPY package.json /app
+RUN npm install
+COPY . /app
+CMD npm run build && node ./build/run.js
```

Comment:
There is a difference between CMD and RUN (https://goinbigdata.com/docker-run-vs-cmd-vs-entrypoint/)

You want rather run something along the lines of 
```
RUN npm install
# If you are building your code for production
# RUN npm install --only=production

# Bundle app source
COPY . .

RUN npm run build

CMD node ./build/run.js
```

With your current command it will rebuild the files everytime you want to run the container

Reply:
Updated pull request!